spec/cassettes/court_application_searcher/unauthorised.yml

```diff
@@ -0,0 +1,35 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: "<COMMON_PLATFORM_URL>/applications/00004c9f-af9f-401a-b88b-78a4f0e08163"
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Ocp-Apim-Subscription-Key:
+      - INCORRECT KEY
+      User-Agent:
+      - Faraday v1.0.1
```

Comment:
did you created this cassette?
this is an old faraday version, if you recreate it it should be 2.xx